manager/integration/deploy/test.yaml

```diff
@@ -10,7 +10,7 @@ metadata:
   name: longhorn-test-role
 rules:
 - apiGroups: [""]
-  resources: ["nodes", "nodes/status", "pods", "pods/exec", "persistentvolumes", "persistentvolumeclaims", "secrets"]
+  resources: ["nodes", "nodes/status", "pods", "pods/exec", "persistentvolumes", "persistentvolumeclaims", "persistentvolumeclaims/status", "secrets"]
```

Comment:
Why we need this?

Reply:
i use `read_namespaced_persistent_volume_claim_status` check if the new PVC status is `Bound`

https://github.com/longhorn/longhorn-tests/blob/b3ec4e9e64ce60608d90db638d6116a92232f258/manager/integration/tests/test_basic.py#L1240